### PR TITLE
Issue 1723

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by pmbohm** [comment]
fixes #1723 and future issues

File: grails-app/views/_js_includes.gsp [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 --%>
 
+
+<base href="${grailsApplication.config.grails.serverURL}" >
```
Comment (dnahodil) [inline grails-app/views/_js_includes.gsp]: Is this needed? Removing it didn't seem to break anything for me.

File: grails-app/views/header/_mainPortalHeader.gsp [hunk 2]
```diff
@@ -9,8 +9,8 @@
 <div id="header">
 
     <div id="logoContainer">
-        <a href="landing"><img src="${portalBranding.logoImage}" alt="logo" id="logo" />
-        </a>
+        <g:link controller="landing"><g:img uri="${resource(base: portalBranding.brandingBase, file: portalBranding.logoImage)}" alt="logo" id="logo" />
```
Comment (danfruehauf) [inline grails-app/views/header/_mainPortalHeader.gsp]: I'm not reviewing this PR, but it seems like this logic of link generation should be in `portalBranding` if at all s.t. it returns a fully qualified link from `portalBranding` if needed.

File: pom.xml [hunk 3]
```diff
@@ -332,7 +334,7 @@
                         <include>portal/ui/VisualisePanel.js</include>
                         <include>portal/ui/MainPanel.js</include>
                         <include>portal/utils/moment.min.js</include>
-                        <include>portal/utils/Proxy.js</include>
+                        <!--<include>portal/utils/Proxy.js</include>-->
```
Comment (dnahodil) [inline pom.xml]: If this isn't needed then we should remove it, not just comment it out.

File: src/test/javascript/spec_helper.js [hunk 4]
```diff
@@ -5,19 +5,19 @@
  *
  */
 
+var BASE_URL = "${grailsApplication.config.grails.serverURL}";;
```
Comment (dnahodil) [inline src/test/javascript/spec_helper.js]: Is this one still needed?

**Comment by dnahodil** [comment]
I have made a few small comments, @pmbohm. Nothing major just a few things that could be tidied-up.

However, my big comment/question is this:
I believe these changes fix the problem (I did some testing) however I don't believe the problem is big enough to justify a change which affects so many files and functions of the Portal (every URL we use/make, essentially). If we had found a one-liner in UrlMappings.groovy or an Apache config change which could fix it without having to make changes throughout the Portal then I would be much happier. Would you agree that this is changing a lot of files to fix a relatively small bug, @pmbohm?

**Comment by danfruehauf** [comment]
@dnahodil I'm with you on this one. I suspect that this change can create more problems down the track which we are not aware off. It does indeed fix a relatively small problem overall.

I think we should try and find a smaller one-line fix because it should exist.

**Comment by danfruehauf** [comment]
In addition this change also breaks other things. For instance I have dev.aodn.org.au -> 172.X.X.X. This change makes all the requests go to 127.0.0.1 and breaks everything.

**Comment by danfruehauf** [comment]
Furthermore, also portal 2 suffers from the same:
http://portal.aodn.org.au/aodn/home
http://portal.aodn.org.au/aodn/home/

**Comment by dnahodil** [comment]
This issue has been fixed with #1746

**Pull request closed without merging**